packages/react-native/local-cli/runMacOS/runMacOS.js

```diff
@@ -278,9 +282,15 @@ module.exports = {
   options: [
     {
       name: '--configuration [string]',
-      description: 'Explicitly set the scheme configuration to use',
+      description:
+        '[Deprecated] Explicitly set the scheme configuration to use',
       default: 'Debug',
     },
+    {
+      name: '--mode [string]',
+      description:
+        'Explicitly set the scheme configuration to use. This option is case sensitive.',
+    },
     {
       name: '--scheme [string]',
       description: 'Explicitly set Xcode scheme to use',
```

Comment:
Based on the description, how does `--mode` differ from `--scheme`?

Reply:
Changed, should be more clear now.